models/catogories.model.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection");
+
+exports.selectAllCategories = () => {
+
+    return db.query(`SELECT * FROM categories;`)
+
+    .then(({rows}) =>{
+        if (rows.length === 0){
```

Comment:
We do not need to reject the promise here because if the database was empty we'd send an empty array 🙂